section/session-flow.html

```diff
@@ -1,54 +1,216 @@
+  </p>
+
+  <p>
+    If any of the conditions defined in the
+    <a href="https://webmonetization.org/specification/#link-type-monetization">link fetch and processing timing</a>
+    cause a monetization link element to be reprocessed, the user agent <em class="rfc2119">MUST</em> [=end session=] for the session
+    associated with that element (if any) and then [=initiate a session=] for that element.
+  </p>
+
+  <p>
+    The user agent maintains an [=active sessions list=] to track all currently running monetization sessions.
+    Each session in this list corresponds to a monetization link element and contains the information needed
+    to distribute payments to the website's wallet.
+  </p>
+
+  <dl>
+    <dt><dfn>active sessions list</dfn></dt>
+    <dd>
+      A user agent maintained ordered list of {{Session}} records created by [=initiate a session=].
+    </dd>
+  </dl>
+
+  <h3>Session initiation</h3>
+  <p>
+    This section defines the algorithms for initiating payment sessions at various scopes—document-wide,
+    media-scoped, or for individual monetization link elements. Session initiation discovers website wallet details,
+    negotiates authorization, creates an incoming payment, and registers a session record in the [=active sessions list=].
+  </p>
+
+  <div class="algorithm">
+    <p>To <dfn>initiate document-wide sessions</dfn> given a |document:Document|, run these steps:</p>
+    <ol>
+      <li>If the result of [=get user wallet=] is null, then return.</li>
+      <li>[=initiate sessions scoped to=] the [^head^] element of |document|.</li>
+      <li>For each [=child navigable=] of |document|, recursively [=initiate document-wide sessions=].</li>
+    </ol>
+  </div>
+
+  <div class="algorithm">
+    <p>To <dfn>initiate media-scoped sessions</dfn> given a |media:HTMLElement|, run these steps:</p>
+    <ol>
+      <li>If the result of [=get user wallet=] is null, then return.</li>
+      <li>[=initiate sessions scoped to=] |media|.</li>
+    </ol>
+  </div>
+
+  <div class="algorithm">
+    <p>To <dfn>initiate sessions scoped to</dfn> the |root:HTMLElement|, run these steps:</p>
+    <ol>
+      <li>Let |linkElements:NodeList| be all [^link^] elements with `rel="monetization"` that are descendants of |root|.</li>
+      <li>For each |linkElement:HTMLLinkElement| in |linkElements|, [=initiate a session=] given |linkElement|.</li>
+    </ol>
+  </div>
+
+  <div class="algorithm">
+    <p>To <dfn>initiate a session</dfn>, given |linkElement:HTMLLinkElement|, perform the following steps:</p>
+    <ol>
+      <li>
+        If the result of [=get user wallet=] is null, then return.
+      </li>
+      <li>
+        Let |document:Document| be the {{Node/ownerDocument}} of |linkElement|.
+      </li>
+      <li>
+        If [=document monetization disabled=] given |document|, then return.
+      </li>
+      <li>
+        Let |websiteWalletAddress:DOMString| be the value of the `href` attribute of |linkElement|.
```

Comment:
Q: Will `href` attribute be resolved URL (`elem.href`) or same as `elem.getAttribute('href')`?

Reply:
I meant this:
<img width="673" height="243" alt="image" src="https://github.com/user-attachments/assets/e3caab89-2ffd-488a-b01f-13c5c133436f" />